docs/sources/tempo/release-notes/v2-4.md

```diff
@@ -0,0 +1,248 @@
+* Localblocks processor honor tenant max trace size limit. [PR 3305](https://github.com/grafana/tempo/pull/3305)
+
+## Upgrade considerations
+
+When [upgrading]({{< relref "../setup/upgrade" >}}) to Tempo 2.4, be aware of these considerations and breaking changes.
+
+### Transition to vParquet 3
+
+vParquet3 format is now the default block format. It is production ready and we highly recommend switching to it for improved query performance and [dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}).
+
+Upgrading to Tempo 2.4 modifies the Parquet block format. Although you can use Tempo 2.3 with vParquet2 or vParquet3, you can only use Tempo 2.4 with vParquet3.
+
+With this release, vParquet is being deprecated. Tempo 2.4 will still read vParquet1 blocks. However, Tempo will fail to configure them. [[PR 3377](https://github.com/grafana/tempo/pull/3377/files#top)]
+
+For information on upgrading, refer to [Upgrade to Tempo 2.4](https://grafana.com/docs/tempo/latest/setup/upgrade/) and [Choose a different block format](https://grafana.com/docs/tempo/next/configuration/parque#choose-a-different-block-format) .
+
+### Updated, removed, or renamed configuration parameters
+
+<table>
+  <tr>
+   <td>Parameter
+   </td>
+   <td>Comments
+   </td>
+  </tr>
+  <tr>
+   <td><code>autocomplete_filtering_enabled</code>
+   </td>
+   <td>Set to <code>true</code> by default [PR <a href="https://github.com/grafana/tempo/pull/3178">3178</a>]
+   </td>
+  </tr>
+  <tr>
+   <td><code>distributor.log_received_traces</code>
+   </td>
+   <td>Use the <code>distributor.log_received_spans</code> configuration block instead. [PR <a href="https://github.com/grafana/tempo/pull/3008">#3008</a>]
+   </td>
+  </tr>
+  <tr>
+   <td><code>tempo_query_frontend_queries_total{op="searchtags|metrics"}</code>
+   </td>
+   <td>Removed deprecated frontend metrics configuration option
+   </td>
+  </tr>
+</table>
+
+The distributor now returns 200 for any batch containing only `trace_too_large` and `max_live_traces` errors. The number of discarded spans are still reflected in the `tempo_discarded_spans_total metrics`.
+
+### Cache configuration refactored
+
+The major cache refactor to allow multiple role-based caches to be configured. [[PR 3166](https://github.com/grafana/tempo/pull/3166)]
+This change resulted in the following fields being deprecated.
+These have all been migrated to a top level `cache:` field.
+
+For more information about the configuration, refer to the [Cache]({{< relref "../configuration#cache" >}}) section.
+
+The old configuration block looked like this:
+
+```yaml
+storage:
+  trace:
+    cache:
+    search:
+      cache_control:
+    background_cache:
+    memcached:
+    redis:
+```
+
+With the new configuration, you create your list of caches,- with either `redis` or `memcached` cluster with your config, then define the types of data and roles.
+
+```yaml
+cache:
```

Comment:
some of these settings are super esoteric. i would use something simpler like i put in the blogpost:

```
cache:
  caches:
  - memcached:
      host: <some memcached cluster>
    roles:
    - bloom
    - parquet-footer
  - memcached:
      host: <some memcached cluster>
    roles:
    - frontend-search
```

Reply:
Okay. I'll swap out the example with the one you provided.

Reply:
Done. Kept the short example here and the longer example in the upgrade page.